app/models/rejection.rb

```diff
@@ -37,6 +37,9 @@ class Rejection < ActiveRecord::Base
   validates :reasons, presence: true
   validate :validate_allowance_renews_on_date
 
+  before_save do
```

Comment:
Yes I did think about that, I preferred cleaning up the ```reasons``` array before saving as I had very little idea how that would impact the aggregations calculations for the stats and for what it is worth didn't want to spend ages figuring out what are the implication when i could just simply user `Array#uniq!`, seemed more pragmatic.